projects/packages/connection/src/class-tokens.php

```diff
@@ -52,14 +52,18 @@ public function validate( $user_id = null ) {
 			'sites/' . $blog_id . '/jetpack-token-health'
 		);
 
-		$user_token = $this->get_access_token( $user_id ? $user_id : get_current_user_id() );
 		$blog_token = $this->get_access_token();
 		$method     = 'POST';
 		$body       = array(
-			'user_token' => $this->get_signed_token( $user_token ),
 			'blog_token' => $this->get_signed_token( $blog_token ),
 		);
-		$response   = Client::_wp_remote_request( $url, compact( 'body', 'method' ) );
+
+		if ( false !== $user_id ) {
```

Comment:
Hey, Leo!

I'm afraid the WPCOM remote endpoint requires both tokens as request arg.
Here's the corresponding 400 response I get with this:

{"headers":{},"body":"{\"code\":\"rest_invalid_param\",\"message\":\"Invalid parameter(s): user_token\",\"data\":{\"status\":400,\"params\":{\"user_token\":\"user_token is not of type string.\"},\"details\":{\"user_token\":{\"code\":\"rest_invalid_type\",\"message\":\"user_token is not of type string.\",\"data\":{\"param\":\"user_token\"}}}}}","response":{"code":400,"message":"Bad Request"},"cookies":[],"filename":null,"http_response":{"data":null,"headers":null,"status":null}}

In this case the test is marked as skipped (this is legit) to not interfere with the actual results from the token health endpoint.

This is why it's not so easy to observe.
<img width="1053" alt="Screenshot 2021-03-25 at 4 35 45 PM" src="https://user-images.githubusercontent.com/1758399/112490601-2f166e80-8d88-11eb-9d7d-602d87d5c44b.png">